packages/components/src/resizable-box/style.scss

```diff
@@ -1,16 +1,12 @@
 .components-resizable-box__handle {
 	display: none;
+	width: $resize-handler-container-size;
+	height: $resize-handler-container-size;
 
 	// Show the resize handle when selected.
 	.components-resizable-box__container.is-selected & {
 		display: block;
 	}
-
-	// The handle is a pseudo-element and will sit inside this larger
```

Comment:
Did we lose this comment? Is it no longer needed?

Reply:
Thanks for noticing, I've added a new comment above the selector to explain the situation.